src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java

```diff
@@ -26,6 +23,11 @@ public static class General {
 		public final static double VOLTAGE_COMP_VAL = 11.5;
 		public final static double RAMP_RATE_VAL = 0.4;
 		
+		public static class Field{
+			public final static double fieldLength = 16.54175;
```

Comment:
Raz - consts are UPPER_SNAKE_CASE

Reply:
we'll fix it on another time to hopefully avoid conflicts